package.json

```diff
@@ -105,6 +105,7 @@
     "promise": "^8.0.3",
     "prop-types": "^15.7.2",
     "react-devtools-core": "^4.6.0",
+    "react-native-codegen": "^0.0.7",
```

Comment:
Note: On 0.64-stable branch, React Native will always use CodeGen from `npm`, even when running from RNTester. 

I wish we had Yarn Workspaces set up right, that way, I could point here to use local version of CodeGen when running from this repository.

Something we will have to improve in order to merge this to master.

CC: @hramos

Reply:
In this particular case, this is more than fine. Right now, on 0.64-stable, in `packages`, we have CodeGen at 0.0.6 which doesn't work with this release. We would need to cherry-pick ALL commits related to CodeGen to bump it to 0.0.7.

By using an npm version instead, we can test RNTester and others. This isn't ideal setup, but works.

Reply:
Can you use `repo-config/package.json` to add the codegen dep? This one is a regular workspace.